sdk/include/token.h

```diff
@@ -8,8 +8,8 @@
 #include <stdlib.h>
 #include <timeout.h>
 
-struct SKeyStruct;
-typedef struct SKeyStruct *SKey;
+struct TokenKeyType;
+typedef struct TokenKeyType *TokenKey;
```

Comment:
For backwards compatibility, can we also add a `typedef TokenKey SKey`.  Not sure if you can put `__attribute__((deprecated("SKey has been renamed TokenKey"))` on a type.  If you can, that would be nice as well.